app/controllers/sites_controller.rb

```diff
@@ -12,11 +12,13 @@ def show
 
   # GET /sites/new
   def new
+    @add_sites = Site.all - @site_program.sites
```

Comment:
I don't understand where @site_program is coming from

Reply:
@site_program is coming from [class Programs::SitesController < SitesController]